networking/firewall/000infralib/src/lib/keys/network_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEogIBAAKCAQEAxAjunhwI+cxVZFRLigJ4+q7EJIJ4mOkit00M1ui2JRjoYvJh
+RD37Dau6V+ebl44bqmO8bBlnm+obSFUbaNgDq1p+7c2kAPHI0GtGrjp2aEBVaGnG
+9oBVR+WO8/BgiIQri6HiQ32TM59G/ytBwrM/2kzC7/ArcY5QHeAxewvjoJ9qzuy6
+hkw9fxkM69bA9MRc/6hnJ/4pPwPogKVLjONj5XFWp9W6GAzBEDdVPAWSo+pijvxS
+CoLCQekEQMxniJedf/mpE3+cQYwaTgIMWGiJ9uJxnUOc3lPMByR4YDfwdezJiItO
+lbSphnB0pQ8tliF5hF6qVffD/uxT9OZQYuEcYwIBIwKCAQAyaLJjK8fLNIr8irRW
+sC28zdqqTW93xuurejaAZ7nPBmekajZEv3sSJNglO45ozMVJElUF2qWdHvESmYq0
+j1Fmk5z7UiLbq+MuR3iTM5rKWa+P4K+BNvFboW3m9KsbyjcVRuJpIEpsXCgy/H6f
+x7FVZDIgcPU6dQ1JgsrlIFB5wiOfguX3Cp9NID5jsECE4Gk/hfjlNLhrEgVPHJ8N
+SIxqRvlaie33K6nDh4/bJDgseIPbfIuPqHVXIX2B7cdgA6b4YX9M61BvgkWHwNiy
+ioOB57q87+NaHtdRJco5ZL4Pk/3CepTAij1y3mBqnrDxmYpp7M4b+feh6rovs+ym
+rHaTAoGBAP8dEo4oPsK47Wlg9JUSUf0KGx3hRaY8Sh3XgB+8kABHNEufEZBHagLq
+hereG6sK5Y53EnKkYH//My92qVVldZNMiajC78n23gCXgVHQv4+lW3bnpHwqfYvl
+WqFPrmv+9yIuNRvKeSTv9ohelfecaJplos3XPxvGr9zSr+M2Ay2lAoGBAMS3Tuby
+q0godVuVbaLBQo3FoLo9JGB+yQtZCzMOciUdYev+ARqdo5u7hckmJCmWvmGJd07i
+IV1+FyWX+Cc792RtgxFB4EQnQbX76Bp8ABdYVySBiHOqs6V0b8SjHi+PtiHXztPe
+d0BJiucZnkz41Yv0lLr1esjCjzFMQdTdlJNnAoGBAKelY/cEgQOeGFsxFcEMCf4N
+9I/6b5kgTfZaaiN76ZnIZDGxrHTBNwk7B4u2hzXiltn2edZOwxJInfqeb0a/BB73
+yC0SYw+w2we0BIY4uGW1zmQUkKldwDdcND4ea05IdoQs/lQXVuxjLPqHTJtmyGV1
+/UVwMMkcKmyKc5VPYSyfAoGBAJfAm/P9B8nzUzgFjxcnXzoqvdF/n7gnSqJaoj1U
+SW0WqqAbt7VySwMbobht4WHpUQlqC5Sf0JiUeEGSfZqb+VwZ92UVj7+L/33CU/cs
+dRlSxuGlwQjFg0UfTugmD/jN8uboTxh4ap9OrPtriL8JGcO8rT/Ery0oX9WSmS8u
+ly/hAoGADMnCPotldX5sVIZOHEPxOH5gtZGAkCz2iloOxtC1YmykqAWWEZV9sds/
+aqQ7C7i4t5wD3gySwnFgLVzo402soetbVYnwDY1cr51bSK7HdMGScK7wcLZO53/1
+3eWS4ys8k0GuPitwmFtbYk8/R0OSwEVX/BNEnGKTtX5p6YqJ+uk=
+-----END RSA PRIVATE KEY-----
```

Comment:
@idorax this is a private key and needs to be removed immediately, also may want to tell them to remove this key internally.

Reply:
Hi @rasibley, I have removed the keys. Will send email to the maintainers to remove this key internally later.

Reply:
Thank you @idorax !